### PR TITLE
Rename to Fortran.h, update style guide, run clang-format on everything

### PR DESCRIPTION
**Pull request by klausler**
Address my TODOs from last week's meeting on style and naming.

### CODE REVIEW

File: lib/FIR/builder.h [hunk 1]
```diff
@@ -133,7 +133,6 @@ struct FIRBuilder {
   BasicBlock *cursorBlock_;
   std::vector<BasicBlock *> blockStack_;
 };
-
```
Comment (schweitzpgi) [inline lib/FIR/builder.h]: I wonder if this noise is because we have different versions of clang-format?

Comment (klausler) [reply lib/FIR/builder.h]: Maybe.  I'm using 6.0.   I don't know what you're using.  If it's newer, then maybe you can tell me, and we can upgrade to it.

Comment (schweitzpgi) [reply lib/FIR/builder.h]: I'm using tip (which is a moving 9.0).
I'll use 6.0 from now on.

Comment (klausler) [reply lib/FIR/builder.h]: 6.0 is getting pretty old.  The newest clang-format binary that I can find here at PGI is a 7.0.  I will try it out on f18 and see how big the resulting patch would be.  If it's not too bad, we should upgrade.

Comment (klausler) [reply lib/FIR/builder.h]: clang-format 7.0 didn't change anything from the 6.0 results.  I'll continue to use it unless somebody can point me to a more recent stable release.

**Pull request merged**